src/main/drivers/pwm_mapping.h

```diff
@@ -36,10 +36,10 @@
 #define MAX_INPUTS  8
 
 #define PWM_TIMER_MHZ 1
-//these three have to be the same because of the ppmAvoidPWMTimerClash functions
-#define ONESHOT_TIMER_MHZ 12
+#define ONESHOT125_TIMER_MHZ 8
+#define ONESHOT42_TIMER_MHZ 24
```

Comment:
We can't run OneShot42 on a 24 MHz. It needs to run on a 12 MHz timer to work with the F4s.
168 / 24 / 2 is not an integer.

We can't run OneShot125 on an 8 MHz timer.
168 / 8 / 2 is not an integer.